example/src/controllers/examples/subsections/FormData.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * A component used to display the underlying data model of a react-formable™ form
+ * (requires highlight.js)
+ */
+export default class FormData extends React.Component {
```

Comment:
I like this. We should move this to `examples/src/components` and call it `JSONViewer.js` or something. We can get syntax highlighting pretty simply by using the `MD` component as what is returned. For example:

```js
render() {
    const text = `
        ${this.propsdata}
    `;
    return <MD text={text} />
}
```

Using highlight directly would be cleaner, but this would be a simple way to get it working (lololol). 

Another really fun component to write would be `FormViewer`. It would take in some markup (or a class) and automatically render a json viewer component next to it.